holoviews/operation/element.py

```diff
@@ -575,7 +575,7 @@ class decimate(ElementOperation):
        The x_range as a tuple of min and max y-value. Auto-ranges
        if set to None.""")
 
-    def _process(self, element, key=None):
+    def _apply(self, element, key=None):
```

Comment:
Because it still doesn't handle this case, and changing it would not be backward compatible. It's badly named is all.